content/en/openpnp/FTP/setting-up-the-board/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Setting up the FTP Board"
+linkTitle: "Setting up the FTP Board"
+weight: 3
+type: docs
+description: >
+  Setting up the FTP Board
+---
+
+Note: it is not required that you mount your FTP board in the exact same location as described below. If you follow these steps you will have a slightly easier time setting the location of the board in OpenPnP, but if you need to set up a board somewhere else on the staging plates, that is totally fine, too.
+
+## Securing the FTP Board
+<!-- TODO: Needs real life images -->
+1. Loosely screw in the universal mounting components in the following locations:
+   1. A tabbed holder in $$
+   2. A flat holder diagonally in $$
+   3. A flat holder horizontally in $$
+
+2. Cut strips of double-sided tape and apply them to the top of the FTP board. Do not cover the fiducial markers, but make sure that each of the pads are covered by tape. This will let us mock placing components without using messy solder paste.

```

Comment:
second-person, active voice:
```suggestion
2. Cut strips of double-sided tape and apply them to the top of the FTP board. Do not cover the fiducial markers, but make sure that each of the pads are covered by tape. The double-sided tape lets the components stick to the board so that you can do a "mock" run of a placement job without dealing with solder paste.
```